app/src/main/res/values/strings.xml

```diff
@@ -366,22 +366,22 @@
     <string name="keyboard_theme_title">Keyboard theme</string>
     <string name="keyboard_keys_category">Keys</string>
     <string name="keyboard_auto_go_action_title">Auto key action</string>
-    <string name="keyboard_auto_go_action_summary">Action of the "Go" key performed automatically after pressing a "Field" key</string>
+    <string name="keyboard_auto_go_action_summary">\"Go\" key action after pressing a \"Field\" key</string>
     <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
     <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="allow_no_password_title">Allow no master key</string>
-    <string name="allow_no_password_summary">Enable the \"Open\" button if no credentials are selected</string>
+    <string name="allow_no_password_summary">Shows \"Open\" button if no credentials are selected</string>
```

Comment:
Not the same meaning, deactivating a button does not mean hiding it

Reply:
What exactly is being explained here?

Reply:
That it is now shown does not necessitate it being hidden before.
What does the "Open" button do?